src/renderer/services/bitcoin/ledger.ts

```diff
@@ -18,10 +22,53 @@ const retrieveLedgerAddress = (network: Network) =>
     catchError((error) => Rx.of(RD.failure(error)))
   ).subscribe(setLedgerAddressRD)
 
-const createLedgerService = (): LedgerService => ({
+export const createLedgerService = (): LedgerService => ({
   ledgerAddress$,
   retrieveLedgerAddress,
   removeLedgerAddress: () => setLedgerAddressRD(RD.initial)
 })
 
-export { createLedgerService }
+export const createLedgerTransferService = <Client extends XChainClient, T extends XChainClient$<Client>>(
```

Comment:
No need to create  another factory to have a `LedgerTransferService`, which seems very similar to `createTransferService` (see next comment).